test/toolchainstatus.go

```diff
@@ -10,19 +10,50 @@ import (
 
 type ToolchainStatusOption func(*toolchainv1alpha1.ToolchainStatus)
 
+func NewToolchainStatus(options ...ToolchainStatusOption) *toolchainv1alpha1.ToolchainStatus {
+	toolchainStatus := &toolchainv1alpha1.ToolchainStatus{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      configuration.DefaultToolchainStatusName,
+			Namespace: test.HostOperatorNs,
+		},
+	}
+	for _, apply := range options {
+		apply(toolchainStatus)
+	}
+	return toolchainStatus
+}
+
+func EmptyToolchainStatus() *toolchainv1alpha1.ToolchainStatus {
+	return &toolchainv1alpha1.ToolchainStatus{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      configuration.DefaultToolchainStatusName,
+			Namespace: test.HostOperatorNs,
+		},
+	}
+}
```

Comment:
I guess that there is no need for this function - we can just simply call `NewToolchainStatus()`

Reply:
yes, I removed it in 4dd4023